Documentation.app/Contents/MacOS/Patterns.playground/Pages/Creating nested components.xcplaygroundpage/Contents.swift

```diff
@@ -1 +1,164 @@
+// nef:begin:header
+/*
+ layout: docs
+ title: Creating nested components
+ */
+// nef:end
+import SwiftUI
+import Bow
+import BowEffects
+import BowOptics
 import BowArch
+/*:
+ # Creating nested components
+ 
+ Usually, you will need to create new components in terms of others with lower granularity. This page covers the most important aspects you need to consider when composing different components.
+ 
+ ## Creating a parent view
+ 
+ As components conform to `View`, nothing stops you from adding them to your SwiftUI views. Therefore, assuming you already have a `ChildComponent`, you can easily add it to your view hierarchy:
+ */
+// nef:begin:hidden
+struct ChildDependencies {}
+struct ChildState {}
+enum ChildInput {}
+struct ChildView: View {
+    var body: some View {
+        EmptyView()
+    }
+}
+typealias ChildComponent = StoreComponent<ChildDependencies, ChildState, ChildInput, ChildView>
+
+struct ParentState {
+    let childState: ChildState
+    
+    static var childLens: Lens<ParentState, ChildState> = Lens(
+        get: { $0.childState }, set: { ParentState(childState: $1) })
```

Comment:
```suggestion
        get:  \.childState, set: { ParentState(childState: $1) })
```

Just as suggestion...